src/index.js

```diff
@@ -136,7 +136,7 @@ export default class StepWizard extends PureComponent {
         return this.getSteps().length;
     }
 
-    getSteps = () => this.props.children.filter(el => el);
+    getSteps = () => React.Children.toArray(this.props.children).filter(el => el);
```

Comment:
I'm not sure if this `filter(el => el)` is still needed. To my current understanding, it's useful for removing `null` nodes, but `React.Children.toArray` might already do this.

Reply:
I'd prefer if we didn't have to do a 2nd iteration to remove null nodes. Can you write a unit test with some null children to validate if we still need the `filter`?

Reply:
From my findings, it's not needed anymore. See 06debaa.

Reply:
Great! Let’s merge!